packages/core/src/auth/ui/vue/types.ts

```diff
@@ -35,4 +35,5 @@ export const authFormTelemetryMapping: {
     identityCenterCodeCatalyst: { featureType: 'codecatalyst', authType: 'iamIdentityCenter' },
     identityCenterExplorer: { featureType: 'awsExplorer', authType: 'iamIdentityCenter' },
     aggregateExplorer: { featureType: 'awsExplorer', authType: 'other' }, // this should never actually be used
+    unknown: { featureType: 'unknown', authType: 'other' }, // this should never actually be used
```

Comment:
post 4/30 we will delete the old auth page stuff, just adding this to unblock compilation.